resources/js/langs/de/index.js

```diff
@@ -4,12 +4,16 @@ import litter from './litter.json';
 import location from './location.json';
 import notifications from './notifications.json';
 import profile from './profile.json';
+import { profile_dashboard } from './profile_dashboard/index';
+import { teams_dashboard } from './teams_dashboard/index';
 import { settings } from './settings/index';
 import { locations } from './locations/index';
 import { home } from './home/index';
 import { auth } from './auth/index';
 import signup from './signup.json';
 import tags from './tags.json';
+import litter_presence_comp from './litter_presence_comp.json';
+import del_image_comp from './del_image_comp.json';
```

Comment:
what is _comp? We should use clearer key names eg `profile`, `teams`, `locations`